tests/e2e/e2e_test.go

```diff
@@ -297,8 +298,49 @@ var _ = Describe("odoe2e", func() {
 				curProj = strings.TrimSuffix(curProj, "\n")
 				// Sleep until status tags and their annotations are created
 				time.Sleep(15 * time.Second)
+
+				ignoreFilePath := tmpDir + "/nodejs-ex/.odoignore"
+
+				if createFileAtPathWithContent(ignoreFilePath, ".git\ntests/\nREADME.md") != nil {
+					fmt.Printf("the .odoignore file was not created, reason %v", err.Error())
+				}
+
 				runCmdShouldPass("odo create " + curProj + "/nodejs nodejs --local " + tmpDir + "/nodejs-ex")
 				runCmdShouldPass("odo push")
+
+				// get the name of the pod
+				podName := runCmdShouldPass("oc get pods | grep nodejs | awk '{print $1}' | tr -d '\n'")
+
+				// verify that the views folder got pushed
+				runCmdShouldPass("oc exec " + podName + " -- ls -lai /opt/app-root/src | grep views")
+
+				// verify that the tests was not pushed
+				runCmdShouldFail("oc exec " + podName + " -- ls -lai /opt/app-root/src | grep tests")
+
+				// verify that the README.md file was not pushed
+				runCmdShouldFail("oc exec " + podName + " -- ls -lai /opt/app-root/src | grep README.md")
+
+				// remove the .odoignore file
+				if os.Remove(ignoreFilePath) != nil {
```

Comment:
@mik-dass  Cool!!!
Write the file remove function in util.go and call that function in the test file.

Advantage of doing this
1. Test file will be clean and easy to understand 
2. Same function can be used by multiple test file

Reply:
Why?  This is just a single function call, it will look exactly the same.

Reply:
My concern is to make test file easy to understand i mean in a talkative way

For example 
1. 
```
status := removeFile(ignoreFilePath)
Expect(status).Should(BeTrue())
```
2.
Or you can write the same code in more understandable ginkgo way
```
Expect(os.Remove(ignoreFilePath)).To(BeNil())
```

Example 2 would be more reasonable IMO as a single function call